otherlibs/fiber-lwt/fiber_lwt.mli

```diff
@@ -0,0 +1,8 @@
+(** Lwt integration for fibers *)
+
+module Fiber_inside_lwt : sig
+  val run : 'a Fiber.t -> 'a Lwt.t
+
+  (** Should only be called inside a [run] *)
+  val callback_to_lwt : (unit -> 'a Lwt.t) -> 'a Fiber.t
```

Comment:
What about making the returned fiber return a result? I think that reflects lwt semantics a bit more

Reply:
Seems fine by me. Do you want to do it? You are going to be the main user of this lib, I just wrote it as a proof of concept.